src/NosCore.GameObject/Networking/ClientSession/ClientSession.cs

```diff
@@ -236,15 +240,15 @@ public void ChangeMap(short? mapId, short? mapX, short? mapY)
                 return;
             }
 
-            ChangeMapInstance(Character.MapInstanceId, mapX, mapY);
+            await ChangeMapInstance(Character.MapInstanceId, mapX, mapY);
         }
 
-        public void ChangeMapInstance(Guid mapInstanceId)
+        public async Task ChangeMapInstance(Guid mapInstanceId)
         {
-            ChangeMapInstance(mapInstanceId, null, null);
+            await ChangeMapInstance(mapInstanceId, null, null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)